### PR TITLE
Add views, bootstrap, edit routes

### PR DESCRIPTION
**Pull request by Alice932**

### CODE REVIEW

File: lesson5/app/controllers/patients_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        @patient = Patient.find(params[:id])
+    end
+
+    def create
+        @patient = Patient.new(patient_params)
+        if @patient.save
+            redirect_to @patient
+          else
+            render :new
+          end
```
Comment (olehvavryniv) [inline lesson5/app/controllers/patients_controller.rb:20]: Please fix the indentation here.

File: lesson5/app/controllers/physicians_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        @physician = Physician.find(params[:id])
+    end
+
+    def create
+        @physician = Physician.new(physician_params)
+        if @physician.save
+            redirect_to @physician
+          else
+            render :new
+          end
```
Comment (olehvavryniv) [inline lesson5/app/controllers/physicians_controller.rb:20]: Indentation.

File: lesson5/app/helpers/patients_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module PatientsHelper
+end
```
Comment (olehvavryniv) [inline lesson5/app/helpers/patients_helper.rb:2]: Looks like you do not need this file.

File: lesson5/app/helpers/physicians_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module PhysiciansHelper
+end
```
Comment (olehvavryniv) [inline lesson5/app/helpers/physicians_helper.rb:2]: This also.

File: lesson5/app/views/patients/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<%= form_with model: @patient do |form| %>
+    <p>
+        Имя<br>
+        <%= form.text_field(:first_name, {:class => 'form-control'} ) %>
```
Comment (olehvavryniv) [inline lesson5/app/views/patients/_form.html.erb:4]: You do not need bracers here. Just `form.text_field(:first_name, class: 'form-control' )`

File: lesson5/app/views/patients/edit.html.erb [hunk 6]
```diff
@@ -0,0 +1 @@
+<%= render "form", patient: @patient %>
```
Comment (olehvavryniv) [inline lesson5/app/views/patients/edit.html.erb:1]: You do not need to pass variable to partial view here. `@patient` will be accessible in form view.

File: lesson5/app/views/patients/show.html.erb [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<h3>
+    <h2>Patient №<%= @patient.id%></h2>
+    <hr>
+    <p><b>Name: </b><%= @patient.first_name%></p>
+    <p><b>Last Name: </b><%= @patient.last_name%></p>
+    <p><b>Age: </b><%= @patient.age%> y.o.</p>
+    <p><b>Email: </b><%= @patient.email%></p>
+</h3>
+<div class="d-flex flex-row ">
+  <%= link_to "Edit", edit_patient_path(@patient), :class => 'btn btn-warning  m-1'%>
```
Comment (olehvavryniv) [inline lesson5/app/views/patients/show.html.erb:10]: Please use the new syntax for hashes. `:class => 'btn btn-warning m-1'` -> `class: 'btn btn-warning m-1'`

File: lesson5/config/routes.rb [hunk 8]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
-  get "/articles", to: "articles#index"
-
+  resources :patients do
+    get "patients" => "patients#index", as: "patients"
```
Comment (olehvavryniv) [inline lesson5/config/routes.rb:3]: Why do you need this? `resources` includes `index` path

File: lesson5/config/routes.rb [hunk 9]
```diff
@@ -1,4 +1,8 @@
...
+  end
+  resources :physicians do
+    get "physicians" => "physicians#index" , as: "physicians"
```
Comment (olehvavryniv) [inline lesson5/config/routes.rb:6]: Same here.

File: lesson5/app/helpers/patients_helper.rb (hunk 3 above)
Comment (Alice932) [reply lesson5/app/helpers/patients_helper.rb:2]: This file was generate automatically  `rails generate controller Patients `

**Review by olehvavryniv — APPROVED** [review]

**Pull request merged**